server/middleware/validationMiddleware.test.ts

```diff
@@ -88,7 +88,7 @@ describe('validationMiddleware', () => {
       expect(next).not.toHaveBeenCalled()
       expect(req.flash).toHaveBeenCalledWith(
         'validationErrors',
-        JSON.stringify([{ field: 'child-name', message: notEmptyMessage }]),
+        JSON.stringify([{ field: 'child[name]', message: notEmptyMessage }]),
```

Comment:
This is a minor change to fix field names of nested validation errors.

By default if a govuk input with name, `name="parentProperty[childProperty]"` has an error message it will assign that error message container an id of, `id="parentProperty[childProperty]"`.

But because the field name we assign the error to is, `parentProperty-childProperty` the anchor link for nested errors doesn't match:


https://github.com/ministryofjustice/hmpps-activities-management/assets/125488090/9b5ec7a0-0993-4a6a-8aa9-5741bf920856

This fixes that:


https://github.com/ministryofjustice/hmpps-activities-management/assets/125488090/26a23919-cd33-4738-91d1-2ac1205f5df1